index.html

```diff
@@ -659,18 +659,18 @@ <h4>Hashing (eddsa-rdfc-2022)</h4>
 
           <ol class="algorithm">
             <li>
+              Let <var>proofConfigHash</var> be the result of applying the
+              SHA-256 (SHA-2 with 256-bit output) cryptographic hashing algorithm [[RFC6234]]
+              to the <var>canonicalProofConfig</var>. <var>proofConfigHash</var> will be
+              exactly 32 bytes in size.
+            </li>
+            <li>
 Let <var>transformedDocumentHash</var> be the result of applying the
 SHA-256 (SHA-2 with 256-bit output) cryptographic hashing algorithm [[RFC6234]]
 to the <var>transformedDocument</var>. <var>transformedDocumentHash</var> will
 be exactly 32 bytes in size.
             </li>
             <li>
-Let <var>proofConfigHash</var> be the result of applying the
-SHA-256 (SHA-2 with 256-bit output) cryptographic hashing algorithm [[RFC6234]]
-to the <var>canonicalProofConfig</var>. <var>proofConfigHash</var> will be
-exactly 32 bytes in size.
-            </li>
-            <li>
 Let <var>hashData</var> be the result of joining <var>proofConfigHash</var> (the
 first hash) with <var>transformedDocumentHash</var> (the second hash).
```

Comment:
"joining" remains a troublesome verb. Since "concatenating" is what is meant, that should be what is said.
```suggestion
Let <var>hashData</var> be the result of concatenating <var>proofConfigHash</var>
(the first hash produced above) followed by <var>transformedDocumentHash</var>
(the second hash produced above).
```